Engine/Fuego/EventQueue.h

```diff
@@ -12,9 +12,10 @@ class FUEGO_API EventQueue
     FUEGO_NON_COPYABLE_NON_MOVABLE(EventQueue)
 
     virtual void Update() = 0;
-    virtual std::shared_ptr<const Event> Front() = 0;
```

Comment:
It breaks API and breaks MacOS. We should use shared_ptr here

Reply:
u meant const Event?

Reply:
fixed